keras_retinanet/utils/eval.py

```diff
@@ -21,9 +21,45 @@
 
 import numpy as np
 import os
-
+import io
+import tensorflow as tf
+from PIL import Image
 import cv2
 
+def TensorboardImage(writer, image, number=0, step=0):
```

Comment:
Why is this in caps?

Reply:
there is no reason. Do you prefer display_tensorboard_image ?